tests/workflows/missing_node_output/tests/test_workflow.py

```diff
@@ -10,4 +10,4 @@ def test_workflow__happy_path():
 
     # THEN we should expect the workflow to succeed with the expected output
     assert terminal_event.name == "workflow.execution.fulfilled"
-    assert terminal_event.outputs == {"final_value": "hello undefined"}
+    assert terminal_event.outputs == {"final_value": "hello "}
```

Comment:
[oos] this is the only piece I feel alittle iffy on, but can be sold that empty string is actually better